tests/forms_tests/tests/test_fields.py

```diff
@@ -947,28 +947,28 @@ def test_booleanfield_1(self):
         f = BooleanField()
         self.assertRaisesMessage(ValidationError, "'This field is required.'", f.clean, '')
         self.assertRaisesMessage(ValidationError, "'This field is required.'", f.clean, None)
-        self.assertEqual(True, f.clean(True))
+        self.assertTrue(f.clean(True))
```

Comment:
Yes, I agree, I was too fast to change them all. I think many of the `assertEqual(True` lines should stay as they are. I will review the patch.